src/lib/fnmpapi.vcxproj

```diff
@@ -3,7 +3,9 @@
   <PropertyGroup>
     <ProjectGuid>{15de7d30-6a0b-47c7-8ddc-1cfc658e8a1e}</ProjectGuid>
     <TargetName>fnmpapi</TargetName>
-    <UndockedType>lib</UndockedType>
+    <!-- Temporary hack to disable linking stage in official OneBranch builds, because it's broken -->
+    <UndockedType Condition="'$(UndockedOfficial)' != 'true'">dll</UndockedType>
+    <UndockedType Condition="'$(UndockedOfficial)' == 'true'">lib</UndockedType>
```

Comment:
I think this should be unnecessary, since the other component was having problems linking VS test code. Let's just try making it a `dll`.